src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -247,25 +296,32 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
           return valueFiatUserCurrency
         })()
 
+        const isAsym = foundPool?.isAsymmetric
         if (isRune && bnOrZero(runePerAsset).isGreaterThan(0)) {
           setRuneCryptoLiquidityAmount(crypto)
           setRuneFiatLiquidityAmount(fiat)
-          setAssetFiatLiquidityAmount(fiat)
-          setAssetCryptoLiquidityAmount(bn(crypto).times(bnOrZero(runePerAsset)).toFixed())
+          // If the pool is asymmetric, there is no asset side deposit
```

Comment:
This is right, but wrong - we *need* these values for balance checks, but may not need to account for them when calculating slippage, or when doing balance checks.
This will rug the auto-rebalancing, a better way to achieve this would be to:
- Always prorate amounts as we're doing currently
- use *virtual* terminology to denote that these amounts exist for rebalancing purposes, but may not necessarily be input amounts
- Programatically use the right amounts when
  - doing balance checks
  - doing RUNE slippage calculation

Reply:
02e31783f2